frontend/packages/console-app/src/components/guided-tours/GuidedTourDrawer.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {
+  Drawer,
+  DrawerPanelContent,
+  DrawerContent,
+  DrawerPanelBody,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  Title,
+} from '@patternfly/react-core';
+import './GuidedTourDrawer.scss';
+
+type GuidedTourDrawerProps = {
+  expanded: boolean;
+  title: string;
+  tourDuration: number;
+  onClose: () => void;
+  guidedTourContent: JSX.Element;
+  children?: React.ReactNode;
+  additionalHeaderContent?: React.ReactElement;
+};
+
+const GuidedTourDrawer: React.FC<GuidedTourDrawerProps> = ({
+  expanded,
+  title,
+  tourDuration,
+  onClose,
+  guidedTourContent,
+  children,
+  additionalHeaderContent,
+}) => {
+  const panelContent = (
+    <DrawerPanelContent className="co-guided-tour-drawer__panel-content">
+      <DrawerHead>
+        <div>
+          <Title size="xl" style={{ display: 'inline' }}>
+            {title}
+          </Title>
+          <span className="text-muted co-guided-tour-drawer__subtitle">
+            {`${tourDuration} minutes`}
+          </span>
```

Comment:
Can you use a `Title` component here as well with a smaller size.

Reply:
done